scripts/run_validator_node.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash` to avoid using the wrong bash on a system with with multiple installed bashes